templates/vault_seal_transit.j2

```diff
@@ -0,0 +1,24 @@
+seal "transit" {
+  address            = "{{ vault_transit_address }}"
+  token              = "{{ vault_transit_token }}"
+  disable_renewal    = {{ '"true"' if vault_transit_disable_renewal else '"false"' }}
+
+  // Key configuration
+  key_name           = "{{ vault_transit_key_name }}"
+  mount_path         = "{{ vault_transit_mount_path }}"
+{% if vault_transit_namespace is defined %}
+  namespace          = "{{ vault_transit_namespace }}"
+{% endif %}
+
+  // TLS Configuration
+{% if vault_transit_tls_skip_verify | bool %}
```

Comment:
IMHO `vault_transit_tls_skip_verify` should not exclude `tls_client_cert` / `tls_client_key` parameters because the transit node could still have `vault_tls_require_and_verify_client_cert` turned on.

Reply:
What values should `tls_client_cert` and `tls_client_key` then have if `vault_transit_tls_skip_verify` is enabled?